setup.py

```diff
@@ -61,10 +61,12 @@ def run(self):
         'Programming Language :: Python :: 3',
     ],
     install_requires = [
-        'traitlets',
-        'jupyter_core',
+        'tornado>=5',
```

Comment:
re-iterating https://github.com/takluyver/jupyter_kernel_mgmt/issues/16 , this requirement makes it "impossible" to use jupyter_kernel_test to test kernels with the same minimum requirements as jupyter.  This causes problems especially on distros which usually have pinned tornado < 5 because of incompatibilities with salt https://github.com/saltstack/salt/issues/51883 . e.g. https://packages.ubuntu.com/bionic/python-tornado

Distros need jupyter_kernel_test to test the kernels to ensure compatibility with jupyter.

The current failures on tornado 4 are fixable.  e.g. https://build.opensuse.org/package/view_file/devel:languages:python/python-flower/backport_run_in_executor.patch?expand=1 is a patch I did to keep flower working on tornado 4.